csuntechlab/iSTART
# STR-37 -- FE: Create Calendar page with "dummy" items listed  (The skeleton for a list)

# Description
Created calendar module based off mock-up.
  
# Testing
1. Log in as steve.
2. Visit the /schedule route. **_Implemented for testing purposes_**.
3. Ensure the calendar meets the original mock-up design.
